Libraries/Azure.psm1

```diff
@@ -797,12 +849,7 @@ Function Create-AllResourceGroupDeployments($SetupTypeData, $TestCaseData, $Dist
 		$EndPointAdded = $false
 		$role = 0
 		foreach ( $newVM in $RGXMLData.VirtualMachine) {
-			if ($newVM.RoleName) {
-				$vmName = $newVM.RoleName
-			}
-			else {
-				$vmName = Get-NewVMName -namePrefix $RGName -numberOfVMs $role
-			}
+			$vmName = $VMNames[$role]
```

Comment:
Get-NewVMName will handle windows VM name. Windows computer name cannot be more than 15 characters long on Azure

Reply:
line 323